Code/Tools/ProjectManager/Platform/Windows/ProjectUtils_windows.cpp

```diff
@@ -186,19 +186,50 @@ namespace O3DE::ProjectManager
                     // First try <project-build-path>/bin/$<CONFIG> and if that path doesn't exist
                     // try <project-build-path>/bin/$<PLATFORM>/$<CONFIG>
                     buildConfigurationPath /= "bin";
-                    if (editorPath = (buildConfigurationPath / AZ_BUILD_CONFIGURATION_TYPE / "Editor").
+                    auto paths = AZStd::vector<AZ::IO::FixedMaxPath>{
+                        buildConfigurationPath / AZ_BUILD_CONFIGURATION_TYPE / "Editor",
+                        buildConfigurationPath / AZ_TRAIT_OS_PLATFORM_CODENAME / AZ_BUILD_CONFIGURATION_TYPE / "Editor"
+                    };
```

Comment:
This should use an AZStd::array, as there is no need for heap memory allocations.

```suggestion
                    AZStd::array paths = {
                        AZ::IO::FixedMaxPath { buildConfigurationPath / AZ_BUILD_CONFIGURATION_TYPE / "Editor" },
                        AZ::IO::FixedMaxPath { buildConfigurationPath / AZ_TRAIT_OS_PLATFORM_CODENAME / AZ_BUILD_CONFIGURATION_TYPE / "Editor" }
                    };
```